applications/post-editing.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Human editing of machine translation output
+---
+
+# Post-editing
+
+Human **post-editing** is the review and correction of machine translation output. When post-editing, the human translator checks the original input and the machine translation output, and fixes errors in the output.
+
+Post-editing started to replace human translation because it is faster and cheaper than the traditional human translation process.
+
+Over the 2010s, with rising machine translation quality, it became the dominant human translation workflow.
+
+### Flavours
+With **light post-editing**, the goal is that readers understand the idea of a text. Light post-editing provides the gist of a message. It is usually aimed at internal communications.
+
+With **full post-editing**, the goal is to match human quality. Clients may request full post-editing services for publishable information.
+
+### Metrics
+**Post-editing distance** is a measure of how much the human translator post-edited the machine translation. It's both a metric for a specific translation, and also used in aggregate.
+
+It can be length-normalized and given as a percentage.
```

Comment:
I only changed "normised" to "normalized" in this file, as we agreed on US spelling.

Reply:
I guess the 2 above sentences can go to the end, because they apply to both metrics.

> Post-editing metrics are used both for a specific translation and in aggregate, at the document, project or workflow level.  They can be length-normalized and given as a percentage or score.